tests/characters/test_crypto_characters_and_their_powers.py

```diff
@@ -51,19 +54,17 @@ def test_actor_with_signing_power_can_sign():
     assert verification is True
 
 
-def test_anybody_can_verify(three_agents):
+def test_anybody_can_verify():
     """
     In the last example, we used the lower-level Crypto API to verify the signature.
 
     Here, we show that anybody can do it without needing to directly access Crypto.
     """
-    token_agent, miner_agent, policy_agent = three_agents
-
     # Alice can sign by default, by dint of her _default_crypto_powerups.
-    alice = Alice(policy_agent=policy_agent, always_be_learning=False)
+    alice = Alice(federated_only=True, always_be_learning=False)
```

Comment:
No policy agent here means this a federated test?

Reply:
Good question, would also like to know the answer.

Reply:
If my understanding is correct - then yes.